library/ibmjava

```diff
@@ -4,17 +4,17 @@ Maintainers: Jayashree Gopi <jayasg12@in.ibm.com> (@jayasg12)
 GitRepo: https://github.com/ibmruntimes/ci.docker.git
```

Comment:
Ah, the failing test is because these commits are on the `main` branch. So this just needs a `GitFetch: ref/heads/main` added here at the top.